src/lib.rs

```diff
@@ -66,15 +68,32 @@ use {
   prometheus_metrics::serve_prometheus,
 };
 
+#[derive(Parser, Debug)]
+struct CmdArgs {
+  #[arg(long, default_value_t = false)]
+  /// if you start multiple lemmy server instances set this to true on all but one of them
+  disable_scheduled_tasks: bool,
+  /// set to false to disable the http server
+  #[arg(long, default_value_t = true, action=ArgAction::Set)]
+  http_server: bool,
+  /// set to false to disable the outgoing federation in this process
+  #[arg(long, default_value_t = true, action=ArgAction::Set)]
+  federate_activities: bool,
+  /// the index of this outgoing federation process. only useful if you want to split federation work into multiple servers.
+  /// the first process has number 1.
```

Comment:
```suggestion
  /// The index of this outgoing federation process.
  /// 
  /// Only useful if you want to split the federation workload onto multiple servers.
  /// Defaults to 1.
```

Not sure I understand when I should set this field to anything else than 1. What happens if I have two processes running with index 1 and 2? What's the difference from index 1 and 1?